docs/UserGuide.md

```diff
@@ -93,10 +94,30 @@ Important warnings that you should keep in mind while using this User Guide.
 ## Quick start
 <IMPROVE BY ADDING SCREENSHOTS AND LINKS>
 
-1. Ensure Java 11 is installed in your computer. If it's not, install it [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
-2. Download the latest ConnectCare.jar <Link here>
-3. Copy the jar file to the folder you want to use as the home folder
-4. Double-click the file to start the application!
+This section allows you to start caring for your clients as soon as possible!
+
+### Installation and Startup Instructions
+1. To use ConnectCare you will require Java 11 to be installed in your computer. 
+   1. If you do not have Java 11, install it [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
+   2. If you have Java installed, check that it is the correct version:
+   
+       Windows or macOS users might find [this guide](https://blog.hubspot.com/website/check-java-verison) useful.
```

Comment:
Maybe split into two sub sections to strucutre it better. Include linux users too